rs/sns/governance/src/governance.rs

```diff
@@ -478,10 +478,15 @@ impl GovernanceProto {
     {
         let deployed_version = self.deployed_version_or_err()?;
 
-        let upgrade_steps = {
-            let cached_upgrade_steps = self.cached_upgrade_steps_or_err()?;
-            cached_upgrade_steps.take_from(&deployed_version)?
-        };
+        let cached_upgrade_steps = self.cached_upgrade_steps_or_err()?;
+
+        let upgrade_steps = cached_upgrade_steps.take_from(&deployed_version).map_err(|_| format!("The currently deployed SNS version is not in the cached_upgrade_steps. You may need to wait for the upgrade steps to be refreshed. This shouldn't take more than {} seconds.", UPGRADE_STEPS_INTERVAL_REFRESH_BACKOFF_SECONDS))?;
```

Comment:
`cargo fmt`

Reply:
Somehow this monster survives cargo fmt, but I broke it up onto two lines